clean.cmd

```diff
@@ -80,12 +80,12 @@ if NOT "%clean_targets%" == "" (
 
 if /I [%clean_tools%] == [true] (
   echo Cleaning tools directory ...
-  rmdir /s /q %~dp0tools >> %cleanlog%dir
+  rmdir /s /q %~dp0tools >> %cleanlog%
 )
 
 if /I [%clean_all%] == [true] (
   echo Cleaning entire working directory ...
-  call git clean %~dp0 -xdf >> %cleanlog%
+  call git clean -xdf -e clean.log %~dp0 >> %cleanlog%
 )
 
 echo Done Cleaning.
```

Comment:
For the most part I was checking for ERRORLEVEL after each of the relevant commands but I seem to have only done that for the msbuild call we should do it for the other locations as well.